docs/synapse/userguides/storm_ref_type_specific.rstorm

```diff
@@ -569,173 +593,178 @@ changes (improvements to the ingest logic, additions to the Synapse data model)
-specific values or variables:
+In addition, predictable guids are a "next best" option in cases where secondary deconfliction is not possible.
+This may be the case if an object's secondary properties are not unique enough, or data sources are sparse and
+you cannot identify a property that is guaranteed to be present across all data sources.
 
-.. storm-cli:: $guid=$lib.guid(my_data_source,2022/09/12,https://www.example.com/my_article.html) [ media:news=$guid ]
+Use **arbitrary guids** when:
 
-.. storm-cli:: $source=my_data_source $published=2022/09/12 $url=https://www.example.com/my_article.html $guid=$lib.guid($source,$published,$url) [ media:news=$guid ]
+- you will never need to deconflict the data, so the guid value doesn't matter.
+- a user needs to manually create a guid node, and the asterisk ( ``*`` ) is the simplest means to do so
+  (e.g., if a ``gen.*`` command or other automation does not exist for the kind of node in question).
 
-**Note** that the same guid value is generated in each of the three examples above.
+There are still some deconfliction risks (as described above) when using the asterisk. However, the "ease
+of use" of the asterisk may be an acceptable trade off in some circumstances. For example, the number of
+user-created nodes is likely to small compared to the volume of data ingested through Power-Ups or other
+automation; and nodes initially created with arbitrary guids may later be "recognized" (and deconflicted
+appropriately) by automation that uses secondary property deconfliction.
 
-.. NOTE::
-  
-  The input to the ``guid`` generator is interpreted as a **structured list,** specifically, a list
-  of string values (i.e., ``(str_0, str_1, str_2...str_n)``). Deconfliction depends on the same list
-  being submitted to the generator in the same order each time.
-  
-  The ``guid`` generator is **not** "model aware" and will not recognize items in the list as having
-  a particular data type or representing a particular property value. That is, Synapse will not set
-  any secondary property values based on data provided to the ``guid`` generator. Any property values
-  must be set as part of the node creation process.
-
-A full discussion of writing ingest code (particularly for Storm packages, services, or Power-Ups) is beyond
-the scope of this User Guide. For more information, see the :ref:`devguide`.
 
 Operations
 ++++++++++
 
-Because guid values are unwieldy to use on the command line (outside of copy and paste operations), guid
-nodes may be more easily lifted by a unique secondary property.
+Because guid values are unwieldy to use on the command line (outside of copy and paste operations), it
+is often easier to lift guid nodes by a unique secondary property.
 
 **Examples:**
 
 Lift an org node by its alias:
 
 .. storm-pre:: [ ou:org=* :alias=choam :name='combine honnete ober advancer mercantiles' ]
-.. storm-cli:: ou:org:alias=choam
+.. storm-cli:: ou:org:alias = choam
```

Comment:
Is "spaces around operators" the style we want to move towards? We definitely have a mix of both ways but it seems like currently we have more instances without spaces around the operators.

Reply:
I don't think we have a strong preference - I've been trying to include spaces recently b/c I think it helps the readability but I don't feel strongly enough about it to go update the entire file (though the type-A in me was tempted). :wink:

Reply:
Yeah they do seem a bit more readable especially with the operators that are more than just `=`, and updating everything does seem a bit much to do all at once.